demo/app/models/example_form_builder/radio_group.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module ExampleFormBuilder
+  class RadioGroup < WizardSteps::Step
+    attribute :favourite_drink, array: true
```

Comment:
Does passing `array: true` here do anything? I couldn't see that it was actually achieving anything when I looked into it.